UNIT_TESTS.md

```diff
@@ -0,0 +1,74 @@
+# Unit Tests
+
+Besides the code of the module itself, it is important to submit in addition a
+suite of tests. They guarantee your add-on remains functional when changes are
+applied to the code on which it depends. Once committed to GitHub, the unit
+tests are run automatically by the CI/CD pipeline every time the code in the
+repository is modified. Notifications are issued if a modification somehow
+impacts your add-on.
+
+## Requirements
+
+The unit tests depend on software that you must install in your system
+beforehand: Subversion and the GRASS development package. On a packaged operating
+system like Ubuntu or Debian Linux this can be achieved with:
```

Comment:
The confusion to me here is that this is for existing addons while it is assuming that the contributor has zero setup. These things also apply for development of addons.

Reply:
I had to install these packages only to run the tests. They are not necessary to run the local version of the add-on on my system (Ubuntu 22.04 with Ubuntu-Gis PPA). I will remove the reference to `subversion` in any case, following Markus' comment above.

Reply:
I modified this section in 0574f7287634ba26a4e334e07f326273b7153311. I left the reference to `grass-dev`, I am able to run and install the add-on without it.

Reply:
grass-dev is needed for C and C++ tools.